ansible/content-servers.yml

```diff
@@ -0,0 +1,180 @@
+- hosts: bananas_fileservers
+  gather_facts: true
+  become: true
+
+  vars:
+    # Our vars, for easy reuse
+    cache_timeout: "1y"  # Basically just as long as nginx runs for
+    bananas_cdn_server: bananas.cdn.openttd.org  # Overriden by host vars
+
+    nginx_shared_config: |
+      location = /healthz {
+        access_log off;
+        return 200;
+      }
+      # Rewrite the first to the second:
+      #   base-graphics/12345678/12345678901234567890123456789012/filename.tar.gz
+      #   base-graphics/12345678/12345678901234567890123456789012.tar.gz
+      # This allows the OpenTTD client to know the name to use for the file,
+      #   while the S3 only knows the md5sum based name.
+      location / {
+        rewrite "^/([a-z-]+)/([a-f0-9]{8})/([a-f0-9]{32})/[a-zA-Z0-9-_.]+\.tar\.gz$" $1/$2/$3.tar.gz break;
+        return 404;
+
+        proxy_pass https://{{ bananas_cdn_server }}/;
+        proxy_hide_header Strict-Transport-Security;
```

Comment:
Missing lines are:
`proxy_ssl_server_name on;` <- to enable SNI
`proxy_set_header Host bananas.cdn.openttd.org;" <- to set the right hostname in the SNI

Otherwise it is trying to connect to AWS with either no SNI or with `bananas-1.cdn.openttd.org`, which is not known.